.github/workflows/daily.yml

```diff
@@ -54,6 +54,48 @@ jobs:
           labels: "dependecies"
       - uses: ./.github/actions/build-teardown
 
+  audit-docker:
+    runs-on: ubuntu-latest
+    env:
+      IMAGES: "diem/validator diem/validator_tcb diem/init diem/tools"
+      TAG: devnet
+      AUDIT_SUMMARY_FILE: /tmp/summary
+    steps:
+      - uses: actions/checkout@v2
+      - name: install trivy as needed
+        run: |
```

Comment:
dev-setup.sh will do this if you use the base image you've updated the dev-setup.sh to create.

Reply:
yep, this was added to the dev-setup and and github base image below.  This was for canary before the base image built and published to Dockerhub.

Reply:
Cool, just checking.